src/sidebar/store/modules/selection.js

```diff
@@ -441,12 +441,18 @@ function focusModeFocused(state) {
 }
 
 /**
- * Returns the username for a focused user or `null` if no focused user.
+ * Returns the `userid` for a focused user or `null` if no focused user.
  *
  * @return {object|null}
```

Comment:
The docstring here is out of date as this returns `string|null` rather than `object|null`.